synapse/cortex.py

```diff
@@ -842,7 +841,7 @@ class Cortex(s_cell.Cell):
                             await layr.setOffset(layr.iden, items[-1][0])
 
             except asyncio.CancelledError: # pragma: no cover
-                raise
+                return
```

Comment:
Since this is the main function for the coroutine, catching and handling CancelledError is ok here

Reply:
Doesn't matter either way  since schedCoro will swallow the cancellederror